opslevel/resource_opslevel_trigger_definition.go

```diff
@@ -80,16 +80,35 @@ func resourceTriggerDefinition() *schema.Resource {
 				Optional:     true,
 				ValidateFunc: validation.StringInSlice(opslevel.AllCustomActionsEntityTypeEnum, false),
 			},
+			"extended_team_access": {
+				Type:        schema.TypeList,
+				Description: "The set of additional teams who can invoke this Trigger Definition.",
+				ForceNew:    false,
+				Optional:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			},
 		},
 	}
 }
 
 func resourceTriggerDefinitionCreate(d *schema.ResourceData, client *opslevel.Client) error {
+	var extended_teams *[]opslevel.IdentifierInput
+
+	extendedTeamAccessArray := getStringArray(d, "extended_team_access")
+	if len(extendedTeamAccessArray) > 0 {
+		my_teams := []opslevel.IdentifierInput{}
+		for _, team := range extendedTeamAccessArray {
+			my_teams = append(my_teams, *opslevel.NewIdentifier(team))
+		}
+		extended_teams = &my_teams
+	}
+
```

Comment:
could this be simplified as

```suggestion
    extended_teams := []opslevel.IdentifierInput{}
	extendedTeamAccessArray := getStringArray(d, "extended_team_access")
    for _, team := range extendedTeamAccessArray {
	    extended_teams = append(extended_teams, *opslevel.NewIdentifier(team))
    }
```

Then below could be changed to...